src/components/Charts/CategoryExpense.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from "chart.js";
+import { Pie } from "react-chartjs-2";
+import transactionsService from "../../services/api/transactionsService.";
+import { useState, useEffect } from "react/cjs/react.development";
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+export const CategoryExpense = () => {
+  const [transactions, setTransactions] = useState([]);
+  const [categories, setCategories] = useState([]);
+  const [sums, setSums] = useState(0);
+  const getTransactions = async () => {
+    try {
+      const response = await transactionsService.getTransactionList();
+      setTransactions(response);
+      const transactionCategories = response.map((item) => item.category);
+      setCategories(
+        transactionCategories.filter((x, i, a) => a.indexOf(x) == i)
+      );
+    } catch (error) {
+      throw new Error("Transaction list could not been shown.");
+    } finally {
```

Comment:
prob some loader should be switched off here ;d

Reply:
Not sure what you mean?